code/modules/mob/living/carbon/carbon_defines.dm

```diff
@@ -46,6 +46,8 @@
 	var/datum/action/ability/activable/selected_ability
 	///carbon overlay layers
 	var/list/overlays_standing[TOTAL_LAYERS]
+	///chestburst state
+	var/chestburst = CARBON_NO_CHEST_BURST
```

Comment:
whys it on carbon though shouldnt this be human

Reply:
from what I saw this is always referenced at the carbon level. Of course normally only a human can actually be infected

Reply:
Then it should be at the human level, carbons entire existence is stupid

Reply:
done